apps/builder/app/builder/features/topbar/publish.tsx

```diff
@@ -554,24 +571,40 @@ const ExportContent = () => {
             Step 3
           </Text>
           <Text color="subtle">
-            Use the{" "}
+            Run this command to publish to{" "}
             <StyledLink
-              href="https://vercel.com/docs/cli"
+              href={DeployTargetDocs[deployTarget]}
               target="_blank"
               rel="noreferrer"
             >
-              Vercel CLI
+              {humanizeString(deployTarget)}
             </StyledLink>{" "}
-            to publish your project on Vercel.
           </Text>
         </Grid>
+
+        <Select
+          fullWidth
+          css={{ zIndex: Number.MAX_SAFE_INTEGER.toString() }}
+          id={id}
```

Comment:
DO we need it at all without label

Reply:
I didn't get it, all the other comments are done now 👍

Reply:
What the reason of using id with Select only. Usually its used for labels but I dont see label

Reply:
Ah, not needed. I was trying different components from the design system. And missed removing it. Removed now